### PR TITLE
Added code so that clicking the Donate button launches IAR donate page

### PR DESCRIPTION
**Pull request by mtkarimi97**
"https://raleighmasjid.org/donate/"

### CODE REVIEW

File: app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt [hunk 1]
```diff
@@ -57,6 +61,16 @@ fun DonateScreen() {
...
+
+    if (onClickLaunchPage.value) {
+        onClickLaunchPage.value = false
+        LaunchDonatePage()
+    }
+}
+
+@Composable fun LaunchDonatePage() {
+    val uriHandler = LocalUriHandler.current
+    uriHandler.openUri("https://raleighmasjid.org/donate/")
```
Comment (z3bi) [inline app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt]: can we put this code directly in the `onClick` closure of the button?

Something like 

```
OutlinedButton(
             onClick = {
                 val uriHandler = LocalUriHandler.current
                  uriHandler.openUri("https://raleighmasjid.org/donate/")
             }
```

Comment (mtkarimi97) [reply app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt]: Unfortunately not, I keep getting an error that I can't call a composable function inside a nonComposable function (onClick), which is why I had to take this weird roundabout route. I tried a few ways but this seemed to be the only option that worked

Comment (z3bi) [reply app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt]: so I think we just need to declare the handler at the top of the `DonateScreen` composable.

```
@Composable
fun DonateScreen() {
    val uriHandler = LocalUriHandler.current

    Column(
```

and then in the `onClick` we can call openUri

```
OutlinedButton(
            onClick = {
                uriHandler.openUri("https://raleighmasjid.org/donate/")
            },
```

Comment (mtkarimi97) [reply app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt]: Yeah that works, I just updated the code and pushed it

**Review by z3bi — APPROVED** [review]

**Comment by z3bi** [comment]
Fixes #6

**Pull request merged**